packages/manager/modules/web-universe-components/src/index.js

```diff
@@ -48,7 +47,6 @@ angular.module(moduleName, [
   wucServiceStatusAction,
   wucString,
   wucTabs,
-  wucV6UiSwitch,
```

Comment:
Removing this directive should be marked as a `BREAKING CHANGE`.

Could you please update the associated commit?

Thanks!